vendor-extra/LiberoPatternsBundle/src/Resources/views/content-meta.html.twig

```diff
@@ -0,0 +1,33 @@
+{%- set attributes = attributes|default({})|merge({class: attributes.class|default([])|merge(['content-meta']) }) -%}
```

Comment:
is this imported from `pattern-library`?

Reply:
Yep.